lupa-core/src/main/kotlin/org/jetbrains/research/lupa/kotlinAnalysis/util/kotlin/KotlinTeamDatabaseConnection.kt

```diff
@@ -0,0 +1,39 @@
+package org.jetbrains.research.lupa.kotlinAnalysis.util.kotlin
+
+import org.jetbrains.research.lupa.kotlinAnalysis.util.GitRepository
+import org.jetbrains.research.lupa.kotlinAnalysis.util.PostgresDatabaseConnection
+import java.time.LocalDate
+
+class KotlinTeamDatabaseConnection : PostgresDatabaseConnection("POSTGRES_KOTLIN_DB_CONFIG") {
+
+    fun updateRepoDate(repo: GitRepository) {
+        conn ?: return
+
+        repo.username?.let { repo.repositoryName?.let { updateRepoDate(repo.username, repo.repositoryName) } }
+    }
+
+    fun updateRepoDate(username: String, repoName: String) {
+        conn ?: return
+
+        val currentDate = LocalDate.now()
+        val query = conn!!.prepareStatement(
```

Comment:
You can use just `conn!!.prepareStatement` since after `conn ?: return` the smart casting will be applied automatically